README.md

```diff
@@ -16,514 +16,35 @@ $ npm install
 $ react-native run-ios
 ```
 
-You can check example screens source code in [example module screens](https://github.com/wix-private/wix-react-native-calendar/tree/master/example/src/screens)
-
-This project is compatible with Expo/CRNA (without ejecting), and the examples have been [published on Expo](https://expo.io/@community/react-native-calendars-example)
-
-## Installation
```

Comment:
Please, leave about section, installation here, in README file

Reply:
and links to component usage also 🙂

Reply:
>and links to component usage also 🙂

Sorry, didn't get this one😅
Should I remove links for: 
- General info
- Basic parameters
- Date marking
- Styling

And put these sections in the README file?

Reply:
Done
All info except components in the README.